.scrutinizer.yml

```diff
@@ -1,12 +1,19 @@
 build:
     nodes:
-        analysis:
+        php74:
             environment:
                 php:
                     version: 7.4
             tests:
                 override:
                     - php-scrutinizer-run
+        php80:
+            environment:
+                php:
+                    version: 8.0
+            tests:
+                override:
+                    - php-scrutinizer-run
```

Comment:
this is based on [their docs](https://scrutinizer-ci.com/docs/build/multiple_test_environment#use-case-testing-different-versions), but I'm not familiar with this tool at all. I think this will only apply after this PR is merged, right?

Reply:
Yes :)